tests/dataset_runner.py

```diff
@@ -61,7 +70,27 @@ def run(self, dataset_fixture, run_name_prefix="test"):
         self.wait_for_envelope_to_be_validated()
         if self.export_bundles:
             self.complete_submission()
-            self.wait_for_primary_and_results_bundles()
+            if run_name_prefix == "scale":
+                # == Scaling Logic ==
```

Comment:
Can you move these comments into a docstring at the top of the function?

Reply:
Sure, I just thought it might be more convenient to put it closer to the code, but now I think the code is a bit polluted by the comments :(